### ISSUE TITLE
Komodo IDE not open (request to open/map URI) python code on remote deebugging

### ISSUE THREAD

**Issue by belonesox** [Komodo/KomodoEdit#3482]
### Short Summary
* I tried to use Komodo IDE 11 for remote debugging of python code (simple-dummy wtf.py with couple of print statements).
* I start debugging with "pydbgp.py -d 10.0.2.2:9000 wtf.py" on remote side.
* Komodo IDE asked for connection, but do not ask to map URI (as usual), also did not try to open readonly script.
* See https://vimeo.com/195094328

I tried to force URI mapping on global or project level - no luck.
Also I see a lot of exceptions in
~/.komodoide/11.0/pystderr.log

### Platform Information

*Komodo Edit or IDE?* 
Komodo IDE, version 11.0.2, build 90813, platform linux-x86_64. Built on Mon Dec 4 10:20:45 20

*Komodo Version?* 
Komodo IDE, version 11.0.2, build 90813, platform linux-x86_64. Built on Mon Dec 4 10:20:45 20

*Operating System (and version)?* 
Linux stas-hp8760w 4.9.41-nrj-desktop-1rosa-x86_64 #1 SMP PREEMPT Tue Aug 8 14:50:57 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
ROSA Desktop Fresh R9 release 2016.1 for x86_64

### Additional Information
https://vimeo.com/195094328
[pystderr.log](https://github.com/Komodo/KomodoEdit/files/1745064/pystderr.log)

**Comment by Naatan** [comment]
Can you reproduce this using `Help > Troubleshooting > Restart in Safe-mode` ?

Please verify that when restarting in safe-mode Komodo should be using the default UI (all your settings would be temporarily reset).

**Comment by belonesox** [comment]
Yes, reproduced. (Of course after restart I have to correct debugging port).
Also reproduced on another PC (with same  OS), and on another remote VM (same OS).

**Comment by belonesox** [comment]
I got it! 
If the file on remote system placed on "/root" [sub]folder — this problem exists. If this file on some other folder — all OK.

**Comment by Naatan** [comment]
So the problem was the folder depth? 2 folders or less caused failures?

**Comment by belonesox** [comment]
No. "/tmp/wtf.py" — OK, "/root/wtf.py" — notOK. Can you reproduce?
I think the problem somehow connected with rights...

**Comment by belonesox** [comment]
No. pydbgp.py always called from user root.
"pydbgp.py -r -d 172.31.1.3:9000 /root/wtf.py" 

$ls -l /root/wtf.py 
-rw-r--r-- 1 root root 41 Feb 21 18:00 /root/wtf.py
$ls -l /tmp/wtf.py 
-rw-r--r-- 1 root root 41 Feb 22 18:56 /tmp/wtf.py

 /tmp/wtf.py — OK
 /root/wtf.py — this bug.

**Comment by Naatan** [comment]
Huh, very strange. I'll target this so we can investigate further.

Fwiw I would advise you not to use root at all, it shouldn't be necessary and it tends to complicate things further, as you have noticed.
